src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to the SCSS file is incorrect. You should link to the compiled CSS file instead. The browser does not understand SCSS files. Change the `href` attribute to point to the corresponding `.css` file.